flat_postprocess/protocol_postprocess.py

```diff
@@ -0,0 +1,37 @@
+"""Python code used to postprocess Flat github action data related to Protocol mappings."""
+import csv
+import os
+import sys
+from urllib.request import urlopen
+
+
+PROTOCOL_MAPPINGS = {}
+
+if __name__ == "__main__":
+    if len(sys.argv) == 3:
+        URL = "https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.csv"
+        oui_textfile = urlopen(URL).read().decode("utf-8")  # nosec: pylint: disable=consider-using-with
+        with open(sys.argv[1], "w", encoding="utf-8") as proto_mappings:
+            proto_mappings.write(oui_textfile)
+
+    with open(sys.argv[1], encoding="utf-8") as file:
+        next(file)
+        reader = csv.reader(file)
+        for row in reader:
+            name = row[0]
+            port = row[1]
+            protocol = row[2]
+            if not port.isnumeric():
+                continue
+            if name and port and protocol:
+                if not PROTOCOL_MAPPINGS.get(name):
+                    PROTOCOL_MAPPINGS[name] = {"port_number": int(port), "protocols": []}
```

Comment:
Wondering if we should try to use a ordered dict here instead, this will avoid unnecessary PR updates if the dictionary is just in a new order with the same data?  See comment on this PR for a similar related issue.
https://github.com/networktocode/netutils/pull/199#discussion_r1095088700

Reply:
The format comes out from iana in numeric order, I think we are ok here.